apps/src/templates/progress/progressStyles.js

```diff
@@ -65,28 +65,25 @@ const statusStyle = {
  * Given a level object, figure out styling related to its color, border color,
  * and background color
  */
-export const levelProgressStyle = (level, disabled) => {
+export const levelProgressStyle = (levelStatus, disabled) => {
   let style = {
     borderWidth: 2,
     color: color.charcoal,
     backgroundColor: color.level_not_tried
   };
 
-  if (disabled) {
-    style = {
-      ...style,
-      ...(!disabled && hoverStyle)
-    };
-  } else {
-    if (level.status !== LevelStatus.not_tried) {
-      style.borderColor = color.level_perfect;
-    }
+  if ((disabled && levelStatus !== LevelStatus.submitted) || !levelStatus) {
```

Comment:
@dmcavoy i added this check to ignore the disable status for submitted levels because i noticed that they weren't being filled in when the lesson group is disabled. however when i rebased i think i saw that you had updated the code to pass `disabled=false`, so maybe this isn't necessary anymore?